taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/biocatalogue_styles.css

```diff
@@ -1,10 +1,20 @@
 @charset "utf-8";
-/**
- * BioCatalogue: app/public/stylesheets/styles.css
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Copyright (c) 2009, University of Manchester, The European Bioinformatics 

```

Comment:
Perhaps don't change the copyright if the file biocatalogue_styles.css actually comes from https://github.com/myGrid/biocatalogue - that means there EBI and Southampton also have joint copyright and we should leave the existing header (if any) in place - add a small header if not.

BioCatalogue is a BSD-3 license https://github.com/myGrid/biocatalogue/blob/master/license.txt  - we can add a  attribution for those files to our LICENSE, see example here:

https://github.com/apache/jena/blob/master/LICENSE#L178